docs/_documentations/troubleshooting.md

```diff
@@ -158,6 +158,43 @@ You might occasionally see projects stuck in the `Starting` or `Stopped` state e
 1. In the **Codewind Explorer** view, right-click your project and select **Build**.
 2. Wait for the project state to return to **Running** or **Debugging** in the **Codewind Explorer** view.
 
+<!--
+Action/Topic: Creating a project and/or Checking the application and build statuses
```

Comment:
I am not completely sure what the action/topic here means, but should this not be only for checking the application status?

Reply:
Sakib, good point. I saw that each of the children topics under the parent topic, `Creating a project` had the same `Action/Topic` so I kept it the same for these newly added topics for consistency. 

Think we should change it up? Or keep the same?

Reply:
I think we should change it to be just `Checking application status` because that is what the document topic is about